From fd051e3de351860b52ff1bbe6cab1fa9c153be76 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Tue, 5 Mar 2024 12:18:40 +0100
Subject: [PATCH] Revert "mkosi: Don't fail on systemd-vconsole-setup.service
 failure for now"

This doesn't seem to fail anymore.

This reverts commit 84c7929cd461f6f1cc2c44c69877b9fd0676c794.

(cherry picked from commit 193fa9d7fe383801bf3ff53e5ee5a2c52107270f)

Related: RHEL-27512
---
 .../mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh    | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/mkosi.images/system/mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh b/mkosi.images/system/mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh
index 076026d097..4011ab930b 100755
--- a/mkosi.images/system/mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh
+++ b/mkosi.images/system/mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh
@@ -3,9 +3,6 @@
 
 rm -f /testok
 
-# TODO: Figure out why this is failing
-systemctl reset-failed systemd-vconsole-setup.service
-
 systemctl --failed --no-legend | tee /failed-services
 
 # Check that secure boot keys were properly enrolled.
